components/ui/header.tsx

```diff
@@ -11,26 +11,28 @@ export default function Header() {
     { label: "Github", path: "https://github.com/trypear/pearai-app" },
   ];
   return (
-    <header className="fixed top-0 z-30 w-full animate-fadein-opacity bg-white-50 md:bg-opacity-40 md:bg-gradient-to-b md:from-white-50 md:backdrop-blur-[2px]">
-      <div className="mx-auto max-w-screen-xl px-4 py-2 sm:px-6 sm:py-4">
-        <div className="flex h-10 items-center justify-between text-xl text-black transition ease-in-out">
+    <header className="fixed top-0 z-30 w-full animate-fadein-opacity bg-white-50 bg-opacity-80 shadow-sm backdrop-blur-[16px]">
+      <div className="mx-auto max-w-screen-xl px-4 py-1 sm:px-6 sm:py-2">
+        <div className="text-md flex h-10 items-center justify-between text-secondary-600 transition ease-in-out sm:text-lg">
           {/* Site branding */}
-          <ul className="flex flex-row items-start space-x-2">
+          <div className="flex flex-row items-start space-x-2">
             {/* Logo */}
-            <Link className="-mr-2" href={"/"}>
+            <Link className="-mt-0.5 sm:mt-0" href={"/"}>
               <PearDarkLogo />
             </Link>
             {/* Nav titles */}
-            {navLinks.map((link, index) => (
-              <Link
-                key={index}
-                className="hover:text-gray-600"
-                href={link.path}
-              >
-                {link.label}
-              </Link>
-            ))}
-          </ul>
+            <div className="flex space-x-2">
+              {navLinks.map((link, index) => (
+                <Link
```

Comment:
```suggestion
                <Link className="hover:text-secondary-400" href={link.path}>
```
now we can remove the 
```js
key={link.label}
```